saleor/graphql/product/mutations/products.py

```diff
@@ -271,6 +272,16 @@ class ProductInput(graphene.InputObjectType):
     seo = SeoInput(description='Search engine optimization fields.')
     weight = WeightScalar(
         description='Weight of the Product.', required=False)
+    sku = graphene.String(description=dedent("""Stock keeping unit. Note: this
+    field is required only used for products without variants."""))
+    quantity = graphene.Int(
+        description=dedent("""The total quantity of this variant availablefor
+        sale. Note: this field is only used for products without variants."""))
+    track_inventory = graphene.Boolean(
+        description=dedent("""Determines if the inventory of this variant
```

Comment:
Determines if the inventory of this **product** [...] - I think using the word "product" in this context will be more understandable.